XU-TechnicalPaper.md

```diff
@@ -265,32 +265,32 @@ Sidenote: The broadcasted orderbooks carry the fee amount of each order in a sep
 3.9. Incentivisation - The Role of XUC
 --------------------------------------
 
-Developers, exchanges and functionaries which contribute resources to the XU ecosystem are key for Exchange Union to get up and running, reach a ‘network-effect’ size and be successful in the long-term. We believe a functioning monetary incentive system is the success factor for future Open-Source projects, maybe even the new decentralized economy as a whole. Bitcoin itself is the best example for a working monetary incentive to build and maintain an ecosystem. Exchange Union creates value and should be self-sustaining.
+Developers, exchanges and functionaries which contribute resources to the XU ecosystem are key for Exchange Union to get up and running, reach a ‘network-effect’ size and be successful in the long-term. We believe a functioning monetary incentive system is the success factor for future Open-Source projects. Bitcoin itself is the best example for a system driven by monetary incentives to build and maintain an ecosystem. Similarly, we believe Exchange Union creates value and should be financially self-sustaining.
 
-Flawed incentivisation is believed to be the main reason why other solutions for connecting exchanges didn’t find adoption yet. Some even ask exchanges to pay hardware or monthly fees. From an exchanges point of view, this doesn’t add up without offering something in return, since the solutions essentially offer an easier way for user funds, and thus trading revenues, to leave the platform. Exchange Union will not charge any fees and more importantly enable the exchanges to not only maintain full trading fee revenue but in almost all cases increase such, as described above. Additionally, different functionaries are rewarded with XUC as follows:
+Flawed incentivisation is believed to be the main reason why other solutions for connecting exchanges didn’t find adoption yet. Some solutions even require exchanges to pay hardware or monthly fees. From an exchange's point of view, this doesn’t add up without offering something in return, since the solutions essentially offer an easier way for user funds, and thus trading revenues, to leave the platform. Exchange Union will not charge any fees and more importantly enable the exchanges to increase revenue with trading fees, see above. Additionally, the different functionaries which are rewarded with XUC are as follows:
```

Comment:
For the second to last sentence: "Exchange Union will not charge any fees and, more importantly, will enable exchanges to increase revenue with trading fees as described above."

Reply:
Done.